New CreateEdit documentation (#7424)
marmelab/react-admin: docs/useEditContext.md

---
layout: default
title: "The useEditContext hook"
---

# `useEditContext`

Whenever react-admin displays an edition page, it creates an `EditContext` to store the record, the submit callback, and other data.

> The `CreateContext` is available to descendants of:

Reviewer: ```suggestion
The `EditContext` is available to descendants of:
```

> - `<Create>`,

Reviewer: ```suggestion
- `<Edit>`,
```

> - `<CreateGuesser>`,

Reviewer: ```suggestion
- `<EditGuesser>`,
```

> - `<CreateBase>`,

Reviewer: ```suggestion
- `<EditBase>`,
```

> All descendant components can therefore access the Create context, using the `useCreateContext` hook. 

Reviewer: ```suggestion
All descendant components can therefore access the Edit context, using the `useEditContext` hook. 
```
